README.md

```diff
@@ -37,22 +37,23 @@ The ably-python client has one dependency,
 
 ## Basic Usage
 
-    from ably.rest import AblyRest
+```python
+from ably.rest import AblyRest
 
-    ably = AblyRest("key_str")
-    ably.time() # returns the server time in ms since the unix epoch
-    ably.stats() # returns an array of stats
+ably = AblyRest("key_str")
+ably.time() # returns the server time in ms since the unix epoch
+ably.stats() # returns an array of stats
 
-    # Channels:
-    # Publish a message to channel 'foo'
-    ably.channels.foo.publish('msg_name', 'msg_data')
+# Channels:
+# Publish a message to channel 'foo'
+ably.channels.foo.publish('msg_name', 'msg_data')
 
-    # Get the history for channel 'foo'
-    ably.channels.foo.history()
-
-    # Get presence for channel 'foo'
-    ably.channels.foo.presence()
+# Get the history for channel 'foo'
+ably.channels.foo.history()
 
+# Get presence for channel 'foo'
+ably.channels.foo.presence()
```

Comment:
Well this is not quite accurate unfortunately, or at least this is not our intended API.  Presence is an attribute of a channel and provides access to the `Presence` object.  So it should perhaps be something more like:
```python
channel = ably.channels.foo
channel.presence.get() # => retrieves the list of members on the presence channel
channel.presence.history() # => retrieves the history of presence events on this channel
```